doc/api/http2.md

```diff
@@ -1171,6 +1174,17 @@ server.on('stream', (stream, headers, flags) => {
 });
 ```
 
+#### Event: 'request'
+<!-- YAML
+added: REPLACEME
+-->
+
+* `request` {http2.Http2ServerRequest}
+* `response` {http2.Http2ServerResponse}
+
+Emitted each time there is a request. Note that there may be multiple requests
+per session. See the [Compatibility API](compatiblity-api).
```

Comment:
I think this is great progress, I would prefer landing this, and then we can check where things can be updated. Having some docs for the compat API would help.